larray/tests/test_la.py

```diff
@@ -2765,6 +2770,16 @@ def test_mean(self):
         sex, lipro = la.axes
         assert_array_equal(la.mean(lipro), raw.mean(1))
 
+    def test_with_axes(self):
+        lipro2 = self.lipro.rename('lipro2').labels = \
+            [l.replace('P', 'Q') for l in self.lipro.labels]
+        sex2 = self.age.rename('sex2').labels = ['Man', 'Woman']
+
+        la = LArray(self.small_data, axes=(sex2, lipro2),
+                    title=self.small_title)
+        la2 = self.small.with_axes((sex2, lipro2))
+        assert_array_equal(la, la2)
```

Comment:
This does not test the title, but I am unsure it should. Let's test it manually/explicitly here for now and see how it goes.